### ISSUE TITLE
Server - Recent security updates blocking Google Analytics

### ISSUE THREAD

**Issue by gggrant** [GCTC-NTGC/TalentCloud#520]
# Description
New headers added for security vulnerabilities (identified by first VA scan) are blocking analytics.

Investigate possible workaround to receive analytics data while satisfying security requirements.

**Comment by Khristophor** [comment]
Seems to be a CORS error:
```
Access to script at 'https://www.googletagmanager.com/gtag/js?id=UA-115747902-1' from origin 'https://talent.canada.ca' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'http://www.googletagmanager.com' that is not equal to the supplied origin.
```

**Comment by gggrant** [comment]
I'm going to remove crossorigin and SRI from the live site now. We have another issue where using Google Analytics with Google Tag Manager could cause issues as well, to be done later.

https://www.upbuild.io/blog/regular-google-analytics-and-gtm/
